argoverse/data_loading/argoverse_tracking_loader.py

```diff
@@ -36,7 +36,14 @@ def _read_city_name(path: str) -> str:
 
 
 class ArgoverseTrackingLoader:
-    def __init__(self, root_dir: str) -> None:
+    def __init__(self, root_dir: str, return_missing_image_as: Optional[str] = None) -> None:
+        """
+        root_dir: directory of the dataset
+        return_missing_image_as: specify what to return when there are image with missing timestamp (default = None).
+                            None        -> None,
+                            'backward'  -> closest previous timestamp,
```

Comment:
this looks good. Is there a specific use case when you think the next or previous image would be preferred?

Reply:
Hmm not sure what'd be the use case, just feel like if we have backward only we should probably have forward only as well, in case they want that.
But... I forgot the closest in either direction case, haha. gonna add that